tests/ard_reprocessed_l1s/dev_go_reprocess.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+# If a number is passed in it is assumed to be the scene limit
+# otherwise the default is 400
+
+set -o errexit
+set -o xtrace
+
+if [[ "$HOSTNAME" == *"gadi"* ]]; then
+	echo If module load breaks check out a clean environment
+	module use /g/data/v10/public/modules/modulefiles
+	module use /g/data/v10/private/modules/modulefiles
+  	
+	module use /g/data/u46/users/$USER/devmodules/modulefiles
+
+	module load ard-scene-select-py3-dea/dev_20230523
+	# module load ard-scene-select-py3-dea/20230522
+
+fi
+
+dry_run=" "
+runard="--run-ard"
+runard=""
+ard_env="/g/data/v10/projects/c3_ard/dea-ard-scene-select/scripts/prod/ard_env/prod-wagl-ls.env"
+index_arg="--index-datacube-env /g/data/v10/projects/c3_ard/dea-ard-scene-select/scripts/prod/ard_env/index-datacube.env"
+base_path="/g/data/xu18/ga/"
+new_base_path="/g/data/xu18/ga/reprocessing_staged_for_removal"
+
+if [ -z "$1" ]
+  then
+    echo "No argument supplied"
+	scenelimitvalue=400
+else
+	scenelimitvalue=$1
+fi
+scenelimit="--scene-limit $scenelimitvalue"
+project="v10"
+pkgdir="/g/data/xu18/ga"
+date=$(date '+%Y%m%d_%H%M%S')
+basedir="/g/data/v10/work/ls_c3_ard/"
+
+# #/* The sed command below will remove this block of test code
+# and generate the production script called go_reprocess.sh
+# sed '/#\/\*/,/#\*\// d' dev_go_reprocess.sh > go_reprocess.sh
+# sed '/#\/\*/,/#\*\// d' landsat_c3/dev_go_reprocess.sh > landsat_c3/go_reprocess.sh
+
+# To det up the dev ODC do ./db_index.sh 
+# run ['dev'|'prod']
+run='dev'
```

Comment:
if we set the default mode to non prod, should we have this setting in line 48 on top so that we reflect the fact that this is a configuration variable?

Reply:
For this test script Iet's keep setting the value close to where it is used.